circle.yml

```diff
@@ -20,6 +21,7 @@ dependencies:
     - mkdir -p $HOME/ds005/out && sudo setfacl -R -d -m group:ubuntu:rwx $HOME/ds005 && sudo setfacl -R -m group:ubuntu:rwx $HOME/ds005
     - mkdir -p $HOME/ds054 && sudo setfacl -d -m group:ubuntu:rwx $HOME/ds054 && sudo setfacl -m group:ubuntu:rwx $HOME/ds054
     - mkdir -p $HOME/docs && sudo setfacl -d -m group:ubuntu:rwx $HOME/docs && sudo setfacl -m group:ubuntu:rwx $HOME/docs
+    - mkdir -p /tmp/fslicense && printf "${FS_LICENSE_CONTENT}" > ${FS_LICENSE}
```

Comment:
As discussed - this would be a more robust way to populate the license for CI https://github.com/BIDS-Apps/freesurfer/pull/36/files#diff-29944324a3cbf9f4bd0162dfe3975d88R25